config/blah.config.template

```diff
@@ -80,7 +80,7 @@ blah_require_proxy_on_submit=
 
 #Path where PBS executables are located 
 # NOTE: this path is computed many times; I worry about the overhead here.  -BB, 12-13-2012
-pbs_binpath=`which qsub 2>/dev/null|sed 's|/[^/]*$||'`
+pbs_binpath=/usr/bin
```

Comment:
I'm confused how this will work for non-default pbs location (which are most of them)